fish_title.fish

```diff
@@ -1,4 +1,19 @@
+# You can override some default title options in your config.fish:
+#     set -g theme_title_display_process no
+#     set -g theme_title_display_path no
+#     set -g theme_title_use_abbreviated_path yes
+
 function fish_title
-  [ "$theme_title_display_process" = 'yes' ]; and echo $_ ' '
-  [ "$theme_title_display_path" != 'no' ]; and prompt_pwd
+  if [ "$theme_title_display_process" != 'no' ]
+    echo $_
+    [ "$theme_title_display_path" != 'no' ]; and echo ' '
+  end
+
+  if [ "$theme_title_display_path" != 'no' ]
+    if [ "$theme_title_use_abbreviated_path" = 'yes' ]
+      echo (prompt_pwd)
```

Comment:
you don't need `echo (prompt_pwd)` here, just `prompt_pwd` … because it echos by default :)